src/pretalx/orga/templates/orga/submission/review.html

```diff
@@ -124,6 +124,20 @@
 
                                 {% include "agenda/includes/submission_resource.html" %}
 
+                                {% if not forloop.last %}<br>{% endif %}
+                            {% endfor %}
+                        </div>
+                    </div>
+                {% endif %}
+                {% if submission.private_resources %}
```

Comment:
Do we want the non-public resources listed separately or in the main list and just marked with the lock symbol?